### ISSUE TITLE
Hi, is there any pytorch version of eeg-tcnet?

### ISSUE THREAD

**Issue by chamwen** [iis-eth-zurich/eeg-tcnet#1]

**Comment by Thoriri** [comment]
Hi!

Since the work was done with Tensorflow and Keras we don't provide any Pytorch version of EEG-TCNet.
But following the network architecture figure, you should be able to reproduce the network topology with some minor modifications to count for differences between TF and Torch.

**Comment by okbalefthanded** [comment]
@chamwen you can find a PyTorch implementation of TCN part in braindecode: 
https://github.com/braindecode/braindecode/tree/master/braindecode/models

**Comment by chamwen** [comment]
Thanks!
> @chamwen you can find a PyTorch implementation of TCN part in braindecode:
> https://github.com/braindecode/braindecode/tree/master/braindecode/models

**Comment by 110lmy** [comment]
你好，该代码中的数据集中只有数据的path，label没有相对应的path？

**Comment by okbalefthanded** [comment]
@110lmy check this notebook, the data labels are created after reading the files.
https://github.com/okbalefthanded/eeg-tcnet/blob/master/eeg_tcnet_colab.ipynb
